mne/viz/_figure.py

```diff
@@ -2295,6 +2074,88 @@ def _toggle_vline(self, visible):
         self.mne.vline_visible = visible
         self.canvas.draw_idle()
 
+    def close_all(self):
+        plt.close('all')
```

Comment:
no.  this will close ALL FIGURES, not just the main data browser and its associated child figs.  See how the `_close` method currently works (all you need to do is close the main data browser window and all the child figs will automatically close with it).  Also not clear why this needs to be public.

Reply:
Right, this doesn't need to be public. This method was just something I added where in the current tests `plt.close('all')` would be called. I can specify this more with `_close` as you suggest.